src/sage/graphs/graph.py

```diff
@@ -5014,19 +5015,22 @@ def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3, induce
 
         TESTS::
 
-            sage: g = Graph()
-            sage: g.add_edges([(0, 1), (0, 2), (1, 2), (2, 3), (3, 4), (3, 5), (4, 5), (6, 5)])
-            sage: h = Graph()
-            sage: h.add_edges([(9, 10), (9, 11), (9, 12), (9, 13)])
+            sage: g = Graph([(0, 1), (0, 2), (1, 2), (2, 3), (3, 4), (3, 5), (4, 5), (6, 5)])
+            sage: h = Graph([(9, 10), (9, 11), (9, 12), (9, 13)])
             sage: l = g.minor(h, induced=True)
             Traceback (most recent call last):
             ...
             ValueError: This graph has no induced minor isomorphic to H !
-
-            sage: g = Graph()
-            sage: g.add_edges([(0, 1), (0, 2), (1, 2), (2, 3), (3, 4), (3, 5), (4, 5), (6, 5)])
-            sage: h = Graph()
-            sage: h.add_edges([(7, 8), (8, 9), (9, 10), (10, 11)])
+            
+            sage: # induced minor does not exist, but minor does 
+            sage: g = Graph([(0, 1), (0, 2), (1, 2), (2, 3), (3, 4), (3, 5), (4, 5), (6, 5)])
+            sage: h = Graph([(9, 10), (9, 11), (9, 12), (9, 13)])
+            sage: l = g.minor(h, induced=False)
+            sage: print("minor exists")
```

Comment:
This is not needed. If the previous line passes without error, we know that we have a minor.